i18n/pt-BR/docusaurus-plugin-content-docs/current/diagram-format.md

```diff
@@ -62,14 +62,15 @@ Cada componente deve ter uma única propriedade "id". Se dois componentes têm o
 a simulação pode não funcionar corretamente.
 :::
 
-Uma lista parcial dos tipos de componentes (por exemplo, [wokwi-led] (peças / wokwi-led)) pode ser encontrada na seção "Referência do diagrama" deste guia. No momento, estamos trabalhando para expandir esta lista. Enquanto isso, algumas das partes também estão documentadas em [Wokwi Elements] (https://elements.wokwi.com).
+Uma lista parcial dos tipos de componentes (por exemplo, [wokwi-led](peças/wokwi-led)) pode ser encontrada na seção "Referência do diagrama" deste guia. No momento, estamos trabalhando para expandir esta lista. Enquanto isso, algumas das partes também estão documentadas em [Wokwi Elements] (https://elements.wokwi.com).
```

Comment:
Build still fails - I think the link should be to `parts/wokwi-led` instead of `peças/wokwi-led`

Reply:
Thanks Uri, today the day was busy, I'll fix it tomorrow! Sorry for the inconvenience!